config/repository.go

```diff
@@ -0,0 +1,271 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"reflect"
+	"sync"
+
+	"github.com/syndtr/goleveldb/leveldb"
+	"github.com/syndtr/goleveldb/leveldb/util"
+)
+
+const (
+	ConfigPrefix string = "config"
```

Comment:
```suggestion
	ConfigPrefix string = "config-"
```

Reply:
I do not think we will have multiple config entries for a host.